doc/JSON_INFO.md

```diff
@@ -162,6 +163,11 @@ Use the `Home` key to return to the top.
 # Introduction
 This document describes the contents of the json files used in Cataclysm: Dark days ahead. You are probably reading this if you want to add or change content of Catacysm: Dark days ahead and need to learn more about what to find where and what each file and property does.
 
+# Navigating the JSON
+A lot of the JSON involves cross-references to other JSON entities.  To make it easier to navigate, we provide a script `tools/json_tools/cddatags.py` that can build a `tags` file for you.  If your editor has [ctags support](http://ctags.sourceforge.net/) then you can use this file to easily jump to the definition of any entity.  For example, in Vim you can jump by positioning your cursor over an id and hitting `^]` (by default).
+
+`cddatags.py` is designed to safely update a tags file containing source code tags, so if you want both types of tag in your `tags` file then you can run `ctags -R . && tools/json_tools/cddatags.py`.
```

Comment:
I am interested in using this but "you can run `ctags -R . && tools/json_tools/cddatags.py`." does not tell me much and I doubt it will mean much to the people that would read JSON_INFO.md. I suggest you move what you wrote here to /doc/DEVELOPER_TOOLING.md or /tools/readme.md and just leave a link to that page here.

Then, please describe what I should use to run the ctags command and if you think that is too elementary, please point to where I can read up on how I would do this.

Reply:
I was torn between here and DEVELOPER_TOOLING.  I'll move this second paragraph there.

For the first paragraph, it really should be as simple as running that script and then figuring out how your editor works with tags files.  I run into trouble providing more details, for a couple of reasons:
* It's hard for me to give reasonable guidance for Windows users; I don't really know what the easiest approach is for that platform, but it's probably the one most in need of it.
* Every text editor is different.  I explained what to do for Vim, but I have no idea what the most popular editors amongst CDDA JSON folks is, nor would I probably know how to configure them.
I'm hopeful that someone who is in a better position to expand on these points will come along and add more detail.